plugin-server/src/worker/ingestion/team-manager.ts

```diff
@@ -51,6 +52,13 @@ export class TeamManager {
             // the team cache every 2min, irrespective of the last access
             updateAgeOnGet: false,
         })
+        this.billingUsageCache = new LRU({
+            max: 10000,
```

Comment:
Problem: If we have more than 10k active orgs at any point then we start doing these queries really rapidly.